src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const main = document.querySelector('body');
+  const element = document.createElement('div');
+  const elemTitle = document.createElement('h2');
+  const elemDescription = document.createElement('p');
+
+  element.style.boxSizing = `content-box`;
+  element.style.top = `${posTop}px`;
+  element.style.right = `${posRight}px`;
+  element.classList.add('notification', type);
+
+  elemTitle.textContent = title;
+  elemTitle.className = 'title';
+
+  elemDescription.textContent = description;
+  elemDescription.style.whiteSpace = 'pre-line';
+
+  element.append(elemTitle);
+  element.append(elemDescription);
+  main.append(element);
+
+  setTimeout(() => {
+    main.remove(element);
```

Comment:
https://user-images.githubusercontent.com/38065883/175120108-5c372d8d-e1b5-46d8-a7bd-15c2ec466f4e.mov

You should remove only messages from the page, not entire page